client/src/components/DatePicker/DatePicker.js

```diff
@@ -2,19 +2,11 @@ import React from 'react'
 import './DatePicker.css'
 import imageFile from '../../images/Search Icon_outlined.svg'
 
-function Child(props) {
-    function handleChange(event) {
-        // Parent calls Child and function onChange
-        props.onChange(event.target.value);
-    }
-  
-    return <input value={props.value} onChange={handleChange} />
-}
 
 const DatePicker = (props) => 
 <div>
 <label className="date-picker" for='requested-date-input'>{props.label}</label>
-<input type="date" className="requested-date-input" name='Requested Date' value={props.value} min={props.min} step="1" max={props.max)/>
+<input type="date" className="requested-date-input" name='Requested Date' value={props.value} min={props.min} step="1" max={props.max}/>
 </div>  
```

Comment:
Can you fix the indentation here?

Reply:
Also, `name` should be set by a prop as well

Reply:
So should `step`

Reply:
Indent repaired and props for all elements added.